### PR TITLE
readme-updates

### PR DESCRIPTION
**Pull request by matschaffer**
Finally got this working so noted down a few more bits to hopefully make it smoother for others in the future.

### CODE REVIEW

**Review by ruflin — APPROVED** [review]
LGTM. It has been a while since I set it up so didn't go through the flow but what you put down here makes a lot of sense.

**Comment by matschaffer** [comment]
Thanks! Looks like I lack a merge button here so will leave it to you or @graphaelli

**Comment by ruflin** [comment]
No merge button for me either 😢

File: README.md [hunk 1]
```diff
@@ -28,23 +28,32 @@ Once tokens have been obtained, `zat -no-server` will perform only archival duti
 ### Credentials
 
 * Obtain Google credentials
-  * [Create an Oauth Client ID credential](https://console.cloud.google.com/apis/credentials) _note: you may need to create a new project first_
+  * [Create an Oauth Client ID credential](https://console.cloud.google.com/apis/credentials)
+    * You may need to create a project, or use a dev project you have access to. If the project doesn't have OAuth consent screen info, you'll need to add that as well.
+      * Choose "internal" user type, give it a name similar to the project name, and add your contact email
+    * Choose "Web application" as the client ID type
+    * Name it "zat"
```
Comment (graphaelli) [inline README.md]:
```suggestion
```

**Review by graphaelli — CHANGES_REQUESTED** [review]

**Review by graphaelli — APPROVED** [review]
Thanks!

**Pull request merged**